galley/pkg/config/mesh/fs.go

```diff
@@ -23,11 +23,9 @@ import (
 	"github.com/gogo/protobuf/proto"
 
 	"istio.io/api/mesh/v1alpha1"
-
-	"istio.io/pkg/filewatcher"
-
 	"istio.io/istio/galley/pkg/config/scope"
 	"istio.io/istio/pkg/config/event"
+	"istio.io/pkg/filewatcher"
```

Comment:
this should be grouped together with `istio.io/api/` L25

Reply:
Now `gci` only support std/remote/local block and use common asc sort order in each block.

Reply:
BTW, `/release-notes-none` does not work, which label is for this?

Reply:
I am not sure either